lib/kubernetes-deploy/renderer.rb

```diff
@@ -16,11 +16,12 @@ def initialize(msg, parents: [], content: nil, filename:)
     end
     class PartialNotFound < InvalidTemplateError; end
 
-    def initialize(current_sha:, template_dir:, logger:, bindings: {})
+    def initialize(current_sha:, template_dirs:, logger:, bindings: {})
       @current_sha = current_sha
-      @template_dir = template_dir
-      @partials_dirs =
-        %w(partials ../partials).map { |d| File.expand_path(File.join(@template_dir, d)) }
+      @template_dirs = template_dirs
+      @partials_dirs = @template_dirs.each_with_object({}) do |template_dir, partials|
+        partials[template_dir] = %w(partials ../partials).map { |d| File.expand_path(File.join(template_dir, d)) }
```

Comment:
This establishes the mapping of `@partials_dir[template_dir] -> [partials_dirs...]`